singularity-eos/eos/eos.hpp

```diff
@@ -154,6 +159,8 @@ class Gruneisen {
   void ValuesAtReferenceState(Real &rho, Real &temp, Real &sie, Real &press, Real &cv,
                               Real &bmod, Real &dpde, Real &dvdt,
                               Real *lambda = nullptr) const;
+  // Vector functions that overload the scalar versions declared here
+  SG_ADD_BASE_CLASS_USINGS(Gruneisen)
```

Comment:
might be worth adding a comment that this contains any generic functions provided by the base class.

Reply:
Done